src/apis/shops.ts

```
@@ -66,9 +66,22 @@ export const searchHandler = async (filters: FilterRequest) => {
     return response.data;
   } catch (err) {
     throw err
   }
+}
+
+export const getDetail = async (shopId: number) => {
+  try {
+    const response = await instance.get(`${shops}/${shopId}`);
+    return response.data;
+  } catch(err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)